packages/manager/modules/netapp/src/dashboard/index/controller.js

```diff
@@ -28,16 +29,29 @@ export default class OvhManagerNetAppDashboardIndexCtrl {
       if (
         this.networkInformations.status === this.NETWORK_STATUS.TO_CONFIGURE
       ) {
-        this.NetAppDashboardService.startNetworkPolling(this.storage).then(
-          ([data]) => {
-            this.NetAppDashboardService.populateStorageNetwork(data).then(
-              (network) => {
-                this.networkInformations = network;
-                this.populateAttachedSubnetAndEndpoint();
-              },
-            );
-          },
-        );
+        this.NetAppDashboardService.startNetworkPolling(
+          this.storage,
+          POLLING_TYPE.ASSOCIATING,
+        ).then(([data]) => {
+          this.NetAppDashboardService.populateStorageNetwork(data).then(
+            (network) => {
+              this.networkInformations = network;
+              this.populateAttachedSubnetAndEndpoint();
+            },
+          );
+        });
```

Comment:
**nitpick(perf):** You need to stop polling as soon as you have the required response. Current implementation keeps making API calls even after the required action is complete and it is a bad implementation.